pydantic_core/core_schema.py

```diff
@@ -2188,7 +2186,7 @@ class UnionSchema(TypedDict, total=False):
 
 
 def union_schema(
-    *choices: CoreSchema,
+    choices: list[CoreSchema],
```

Comment:
This one's docstring needs updating, but I think that's the only place in this file that has a regex hit on `' \*[a-z]'`

Reply:
specifically, here:
```python
    Args:
        *choices: The schemas to match
```
(should change `*choices` to `choices`)